index.html

```diff
@@ -173,13 +173,10 @@ <h3 class="primaryTextAboutMeHeading">About me</h3>
     </section>
     <section class="languageMainContainer">
       <div class="languageSubContainer">
-        <img
-          src="pngFiles/ballIcon.png"
-          alt="languages"
-          width="327px"
-          height="128px"
-        >
-        <li class="languageContainerText">Languages</li>
+        <div class="ellipse">
+          <img src="pngFiles/ballIcon.png" alt="EllipseOne">
```

Comment:
[OPTIONAL]
- [ ] Kindly, make sure that each image in the technology category is centered. This will help in making your project match the Figma design.

- [ ] I also recommend adding proper bottom padding to each technology category title (Skills, Languages, ...).

## Your design:
![image](https://user-images.githubusercontent.com/86069740/236479850-8c122ee6-00f8-4c14-bd35-9192d48c1b4b.png)

## Figma design:
![image](https://user-images.githubusercontent.com/86069740/236480009-f95fa16d-c293-4ba1-bb38-34015613f0ae.png)